apps/io-wallet-user-func/src/app/main.ts

```diff
@@ -56,11 +57,14 @@ const hslJwtValidate = hslValidate(config.hubSpidLogin);
 
 const trialSystemClient = new TrialSystemClient(config.trialSystem);
 
+const pidIssuerClient = new PidIssuerClient(config.pidIssuer);
```

Comment:
You can take the value of `walletProviderEntity` from `config.federationEntity`

```suggestion
const pidIssuerClient = new PidIssuerClient(config.pidIssuer, config.federationEntity);
```